README.md

```diff
@@ -10,12 +10,16 @@ make up this bundle can be found under `charms/`.
 
 ### Setup
 
-If you are on macOS or Windows, you will need to use an Ubuntu VM. You
-can [install multipass][multipass] and access an Ubuntu VM with these
-commands:
-
-    multipass launch --name kubeflow --mem 16G
-    multipass shell kubeflow
+Setting up this bundle requires a Ubuntu VM.
```

Comment:
Could this get changed to `Ubuntu installation or VM`? Just to clarify that it works fine if you've already got Ubuntu installed on your machine.

Reply:
Great suggestion. Just pushed the suggested changes.